plasma_framework/contracts/mocks/framework/registries/ExitGameRegistryMock.sol

```diff
@@ -3,9 +3,15 @@ pragma solidity ^0.5.0;
 import "../../../src/framework/registries/ExitGameRegistry.sol";
 
 contract ExitGameRegistryMock is ExitGameRegistry {
+    uint256 public constant QUARANTINE_PERIOD = 0;
+    uint256 public constant INITIAL_IMMUNE_EXIT_GAMES = 0;
+
     event OnlyFromExitGameChecked();
 
-    function checkOnlyFromExitGame() public onlyFromExitGame {
+    constructor () ExitGameRegistry(QUARANTINE_PERIOD, INITIAL_IMMUNE_EXIT_GAMES) public {
+    }
+
+    function checkOnlyFromExitGame() public onlyFromNonQuarantinedExitGame {
```

Comment:
function name should probably be changed as well: `checkOnlyFromNonQuarantinedExitGame`.

Also, the test description in that uses this should as well be updated: [code pointer](https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/test/src/framework/registries/ExitGameRegistry.test.js#L15)
we should probably add a test in the above place too.